home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1996,30 +2045,8 @@ components:
           items:
             $ref: '#/components/schemas/ReadHouse'
           example:
-            - id: 1,
-              quantity_flat: 45,
-              house_area: 25,
-              adjoining_area: 12,
-              address:
-                region: "Dnipropetrovsk region"
-                city: "Dnipro"
-                district: "Zavodskoy"
-                street: "Kirova"
-                house_block: "2/B"
-                house_number: "23-B"
-                zip_code: "52956"
-            - id: 2,
-              quantity_flat: 60,
-              house_area: 50,
-              adjoining_area: 20,
-              address:
-                region: "Dnipropetrovsk region"
-                city: "Dnipro"
-                district: "Zavodskoy"
-                street: "Kirova"
-                house_block: "2/C"
-                house_number: "24-C"
-                zip_code: "52956"
+            - $ref: '#/components/examples/HouseExample1'
+            - $ref: '#/components/examples/HouseExample2'
```

Comment:
Is this examples rendered properly on web page?

Reply:
Thanks. Rolled back it.